commands/commands.go

```diff
@@ -218,6 +222,28 @@ func determineIncludeExcludePaths(includeArg, excludeArg string) (include, exclu
 	return includePaths, excludePaths
 }
 
+// help is used for 'git-lfs help <command>'
+func help(cmd *cobra.Command, args []string) {
+	if txt, ok := ManPages[cmd.Use]; ok {
+		fmt.Fprintf(os.Stderr, txt)
+	} else {
+		fmt.Fprintf(os.Stderr, "Sorry, no help text found\n")
+	}
```

Comment:
Ah yeah actually you can't use `args[0]` because the 'usage' function which is called for `--help` doesn't include them. I'm using `cmd.Name()` instead which is the external name for `Use` and also dealing with an edge case on the root topic.